e2e/testcafe-devextreme/tests/scheduler/grouping/overflow.ts

```diff
@@ -0,0 +1,80 @@
+import { createScreenshotsComparer } from 'devextreme-screenshot-comparer';
+import Scheduler from 'devextreme-testcafe-models/scheduler';
+import { createWidget } from '../../../helpers/createWidget';
+import url from '../../../helpers/getPageUrl';
+
+fixture.disablePageReloads`Scheduler: Grouping overflow`
+  .page(url(__dirname, '../../container.html'));
+
+['week', 'month'].forEach((viewType) => {
+  ['vertical', 'horizontal'].forEach((groupOrientation) => {
+    ['hidden', 'allDay'].forEach((allDayPanelMode) => {
+      [[9, 14, 60], [0, 24, 360]].forEach(([startDayHour, endDayHour, cellDuration]) => {
+        const allParams = `${viewType} ${groupOrientation} ${allDayPanelMode} ${startDayHour} ${endDayHour}`;
+        test(`Long appointments should not overflow group view (${allParams})`, async (t) => {
+          const scheduler = new Scheduler('#container');
+          const { takeScreenshot, compareResults } = createScreenshotsComparer(t);
+
+          await t
+            .expect(await takeScreenshot(`group-overflow (${allParams}).png`, scheduler.element))
```

Comment:
Minor comment.

It's better to not use the ``` ``` (spaces) because some machines / OS don't handle it properly
But I think it's not a big point in our case :)